lib/register.php

```diff
@@ -460,6 +460,11 @@ function gutenberg_redirect_to_classic_editor_when_saving_posts( $url ) {
 function gutenberg_link_revisions_to_classic_editor( $url ) {
 	global $pagenow;
 	if ( 'revision.php' === $pagenow ) {
+
+		// Only reset the classic editor link.
```

Comment:
Minor: This could be merged into the above condition:

```php
if ( 'revision.php' === $pagenow && ! isset( $_REQUEST['gutenberg'] ) ) {
```

Reply:
yea, good suggestion. i had it that way originally and then thought it might be clearer to break out.